_posts/2020-11-17-quarkiverse.adoc

```diff
@@ -0,0 +1,95 @@
+
+Some have since then created their own repositories external to Quarkus, but others just never happened because not everyone wants the overhead of maintaining a build setup. Even the Quarkus core developers sometimes had ideas for an extension but not solid enough to be included in the Quarkus core repository.
+
+Enter the *Quarkiverse*!
+
+image:/assets/images/posts/quarkiverse/quarkiverse.png[]
+
+Quarkiverse is everything that encompasses the Quarkus universe: that includes the core, platform and all extensions.
+
+When it comes to developing extensions it is simpler to collaborate if the repository structure, the release process and the integration with the existing Quarkus tooling is standardised and automated.
+
+That's where Quarkiverse Hub comes into play.
+
+== What is Quarkiverse Hub?
+
+The link:https://github.com/quarkiverse[Quarkiverse Hub] is our proposed solution to create a "home" for Quarkus extensions maintained by the community.
+
+image:/assets/images/posts/quarkiverse/quarkiverse-repos.png[]
+
+By hosting an extension under the Quarkiverse Hub, the following features are enabled:
+
+Automated and secured publishing of your Maven releases to Maven Central::
+
+Registering and publishing artifacts to Maven central can become a cumbersome task. In Quarkiverse that is automated by GitHub Actions, requiring no manual interaction with any command line tools.
+Releasing an artifact is as simple as opening a pull-request changing the https://github.com/quarkiverse/quarkiverse-template/blob/master/.github/project.yml[`.github/project.yml`] file in the extension's repository.
+Once the pull-request is merged, a GitHub action is triggered invoking `mvn release:prepare release:perform` and the extension is deployed to Maven central.
+
+Automated cross-testing of your extension with Quarkus builds/releases::
+
+Quarkus Core releases happen constantly and keeping up with the latest changes before a release is performed is crucial. Quarkiverse extensions are built daily against the latest https://github.com/quarkusio/quarkus[Quarkus core] using the
+https://github.com/quarkusio/quarkus-ecosystem-ci[Quarkus Ecosystem CI] and listed in the https://status.quarkus.io/[Quarkus Status] page.
+
+Consistent formatting and release of projects::
+
+The `quarkiverse-parent` artifact (inherited by each extension in Quarkiverse) enables plugins that will format your source code to the same format used in Quarkus core.
+
+Documentation aggregation::
+
+Quarkiverse Hub uses https://antora.org[Antora] to aggregate each extension documentation in the Quarkiverse docs website. Check it out at https://quarkiverse.github.io/quarkiverse-docs/
```

Comment:
I would mark this as work in progress too ...not convinced antora actually will work...how is this handled when there is freemarker 1.0 and google-cloud 2.3 ?

Reply:
Antora already handles that, it's just a matter of configuring the `antora.yml` descriptor telling which versions to pick for each repository